packages/core/src/external/aws/lambda-logic/document-uploader.ts

```diff
@@ -1,7 +1,16 @@
 import axios from "axios";
 import { MetriportError } from "../../../util/error/metriport-error";
-import { S3Utils, parseS3FileName } from "../s3";
+import { S3Utils, parseS3FileName, buildDestinationKeyMetadata } from "../s3";
+import { DocumentReference } from "@medplum/fhirtypes";
+import { createExtrinsicObjectXml } from "../../carequality/dq/create-metadata-xml";
+import {
+  METRIPORT_HOME_COMMUNITY_ID,
+  METRIPORT_REPOSITORY_UNIQUE_ID,
+  createPatientUniqueId,
+} from "../../carequality/shared";
+import { Config } from "../../../util/config";
 
+const medicalDocumentsBucketName = Config.getMedicalDocumentsBucketName();
 const api = axios.create();
 
 const UPLOADS_FOLDER = "uploads";
```

Comment:
nit: I believe this is defined in multiple places - can refactor into a single place